### DISCUSSION TITLE
Progress- Completed Kata

### DISCUSSION THREAD

**Discussion by Jonny2Boats** [codewars/codewars.com#3064] [Q&A] [answered]
Sorry to ask a dumb question, but why (on my account anyway) does "Honor Breakdown- Completed Kata"  equal loads (242 in my case), but then in the 'Progress' section, Total Completed Kata equal something weird? (Mine is 57 now?)
I am level 5 overall, and only just levelled up so it is not the number of Kata at current level- to what does it refer?
I am happy to read doc, but all the searches I do show nothing much!
Kindest, J.

**Comment by Madjosz** [comment, marked as answer]
"Honor Breakdown" shows the honor you got from completing kata. The "Progress" stat shows the actual number of completed kata.

You can check the [Codewars docs](https://docs.codewars.com/gamification/honor#solving-kata) on how much honor you gain from which kyu.

**Reply by Jonny2Boats** [reply]
OOOHHHHH!  
OK, thanks for that. Makes sense now!
